### PR TITLE
pe-utils 2.2.0, edge-info and edge-testnet

### PR DESCRIPTION
**Pull request by JanneKiiskila**
# Changes

1. update pe-utils to 2.2.0
   - This brings in `edge-info` and `edge-testnet`.
2. Fix `version`.
    - It was trying to print out `devicedb` version, which is not even part of the build anymore.
    - Fix the prints and add `cosign` version printing.
3. Improvements to GitHub actions to increase test coverage.
   - Keep version information aligned.

Turned out to be a bit trickier than thought due to all of the plugs (access) that had to be set up.

You can run these via:

```
pelion-edge.edge-info
pelion-edge.edge-testnet
```

### CODE REVIEW

**Review by petedyerarm — APPROVED** [review]
LGTM

**Review by jenia81 — APPROVED** [review]

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -877,7 +914,7 @@ parts:
         'bin/tini-static': bin/docker-init
       prime:
         - -bin/tini
-        - bin/docker-init
+        - -bin/docker-init
```
[the remark this answers is not in the document]
Comment (JanneKiiskila) [reply snap/snapcraft.yaml]: Good question. The definition is a fileset, 
https://snapcraft.io/docs/snapcraft-filesets
so, that `- -bini/tini` means that the file bini/tini is NOT included in the prime.

[the remark this answers is not in the document]
Comment (JanneKiiskila) [reply snap/snapcraft.yaml]: However, as we should be the same as with the earlier Docker version - it's actually wrong. It should be like here; https://github.com/PelionIoT/snap-pelion-edge/blob/v1.21.2/snap/snapcraft.yaml#L693-L704

[the remark this answers is not in the document]
Comment (JanneKiiskila) [reply snap/snapcraft.yaml]: Well, turns out it will not work correctly with the the version it used to be, but it works better with what we have.

**Comment by JanneKiiskila** [comment]
CI had a really strange issue, but I think my latest commit should resolve that as well.
- I've never, ever seen that locally (even on fully clean builds).

```
+ git describe --tags
fatal: No names found, cannot describe anything.
Failed to run 'override-build': Exit code was 128.
Error: Process completed with exit code 2.
```

However. using `--always` there should force `git` to come up with some hash rather than fail.

**Pull request merged**